### ISSUE TITLE
Battlscribe crashing when loading CAL

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/horus-heresy#807]
**File:** (HH) Legiones Astartes - Crusade Army List.catz

**BattleScribe version:** 2.00.26

**Platform:** Android

**Dropbox:** No

**Description:** Trying to create any legion astartes roster (even when taken as an ally in other forces) takes 60s of loading time to create the roster, than goes back to the roster menu without having made a new roster. Other armies in the horus heresy repository work fine, it just legion astartes. It seems making a legion astartes roster just crashes the app, the 'creating roster' loading animation also slows down and freezes halfway through before it jumps you back to the menu without a roster

**Comment by zopha** [comment]
Can you check the version of the main Horus Heresy file and the version of the Legiones catalogue? It should be in the name when you go through roster creation.

**Comment by Badgerboym9** [comment]
warhammer 30,000 - horus heresy v64
lergion astartes: age of darkness army list v251

(i posted the OP, had to create account to reply)

**Comment by capitaladot** [comment]
Engaged Low Memory Mode?

**Comment by alphalas** [comment]
That might be it - Android BattleScribe has known issues with low memory mode

**Comment by Badgerboym9** [comment]
I have low memory mode always off. I have tried creating a roster with it both on and off, neither work. I have reinstalled battlescribe, redownloaded the horus heresy files, restarted my phone - nothing has worked. It's the same issue each time, legion astartes wont create rosters, but the others will. I'm going factory reset my phone and retry next

**Comment by alphalas** [comment]
Yeah unfortunately we can't reproduce the issue on android iOS or windows so maybe it is just your device

**Comment by Badgerboym9** [comment]
After the factory reset, it still doesn't work. As a ex-programmer myself this is really baffling me, i can't comprehend what's going on here! I'll try to recreate it on my girlfriends phone, i'll let you know how that goes

**Comment by Badgerboym9** [comment]
After a bit more trying, it nows goes to a black screen and often crashes instead of going back to the roster menu. My android also comes up with a message saying 'storage unavailabe' while the black screen. I've checked my storage and I have 2gb spare. Does the legion astartes lists take up a lot of storage that my phone isn't allowing? If it were on my desktop I'm sure I could fix this, but I'm not sure how to do it on my phone. Any ideas what's wrong with the phone?

**Comment by zopha** [comment]
I haven't come across this before so I'm not sure what the cause might be :(

I pushed a new release last night that removed unneeded conditionals/modifiers for the Army of Dark Compliance, can refresh your data files and see if that helps?  https://battlescribe.net/?faq=refreshData

**Comment by Badgerboym9** [comment]
It still doesn't work but it has changed the error message. After loading for the minute it now comes up with a message saying 'Ran out of data. Low memory mode turned on'. I've checked my memory, Battlescribe uses 20mb average memory use, with a max usage of 128mb and I have 126mb spare memory on the phone. I've tried creating the roster with low memory mode on and off, it hasn't made a difference so far
